force-app/main/default/lwc/affiliationSettings/affiliationSettings.js

```diff
@@ -6,6 +6,7 @@ import { ShowToastEvent } from "lightning/platformShowToastEvent";
 import getAffiliationsSettingsVModel from "@salesforce/apex/AffiliationsSettingsController.getAffiliationsSettingsVModel";
 import getPrimaryAffiliationsSettingsVModel from "@salesforce/apex/AffiliationsSettingsController.getPrimaryAffiliationsSettingsVModel";
 import updateAffiliationMappings from "@salesforce/apex/AffiliationsSettingsController.updateAffiliationMappings";
+import getRecordTypeLabelFromDeveloperName from "@salesforce/apex/UTIL_Describe.getRecordTypeLabelFromDeveloperName";
```

Comment:
I would prefer we not expose UTIL describe any further. If you need a success message, then please have the controller make a call to the record type service to get this information from a model.